### PR TITLE
update base64 from 0.21 to 0.22; zip from 0.6 to 2.x

### PR DESCRIPTION
**Pull request by jqnatividad**
@Kijewski , zip now has new maintainers, and the latest zip has a lot of bug fixes and perf optimizations

cc @Pr0methean

### CODE REVIEW

**Review by Pr0methean — APPROVED** [review]

File: api/src/sign_unsign_zip.rs [hunk 1]
```diff
@@ -25,7 +25,7 @@ where
     let mut input = ZipArchive::new(BufReader::new(input)).map_err(CopyZipError::InputZip)?;
     let mut output = ZipWriter::new(BufWriter::new(output));
 
-    output.set_raw_comment(input.comment().to_owned());
+    output.set_raw_comment(input.comment().to_owned().into());
```
Comment (Kijewski) [inline api/src/sign_unsign_zip.rs:28]: Thank you for the PR! Have you tried if the `.to_owned()` is still needed, or if the `into()` alone might be enough?

Comment (jqnatividad) [reply api/src/sign_unsign_zip.rs:28]: You're right, no need for `.to_owned()` anymore

**Review by Kijewski — APPROVED** [review]
Thank you!

**Comment by Kijewski** [comment]
Released as [v0.1.2](https://crates.io/crates/zipsign-api/0.1.2).

**Pull request merged**